redwood-layout-composeui/src/androidUnitTest/kotlin/app/cash/redwood/layout/composeui/ComposeUiFlexContainerTest.kt

```diff
@@ -68,13 +70,24 @@ class ComposeUiFlexContainerTest(
     private val delegate: ComposeUiFlexContainer,
   ) : TestFlexContainer<@Composable () -> Unit>, FlexContainer<@Composable () -> Unit> by delegate {
     private var childCount = 0
+
     override val children: ComposeWidgetChildren = delegate.children
+
     constructor(direction: FlexDirection, backgroundColor: Int) : this(
       ComposeUiFlexContainer(direction).apply {
         testOnlyModifier = Modifier.background(Color(backgroundColor))
       },
     )
 
+    override fun onScroll(onScroll: (Double) -> Unit) {
+      delegate.onScroll(onScroll)
+    }
+
+    override suspend fun scroll(offset: Double) {
+      delegate.scrollState?.scrollTo(offset.toInt())
+      delay(20.milliseconds)
```

Comment:
Seems like every impl is doing this. Maybe make this non-suspending and move to the test call-site instead?

Reply:
I added it to the method since we can skip the delay on no-op impls like Compose UI/View `LazyList`.